.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -4,15 +4,15 @@ on:
   pull_request:
     branches: [main]
     paths:
-      - 'data_model/**.rs'
-      - 'data_model/**.yml'
-      - 'data_model/**.json'
-      - 'data_model/**.toml'
+      - 'iroha_data_model/**.rs'
```

Comment:
it's one thing to rename crates but why are you renaming directories?

Reply:
Consistency, predictability, less cognitive load

Reply:
I think what we had was fine. Crates that are in the `iroha` repository are prefixed with `iroha` but the prefix is implied for directories. And this will create so many conflicts for PRs

Reply:
This is not true for `iroha`, which had the `client` directory. IMO, it's worth changing and going through a brief patch of conflicts to have a standard naming scheme long-term. It's not uncommon for Rust projects to follow this, e.g. [bevy](https://github.com/bevyengine/bevy/tree/main/crates).

Reply:
I agree that have directories named with full crate names is less confusing and more consistent overall. It is currently a mess - some directories are crates, some are not, some have nested crates. Some crates are named in one way as a directory, but have a different actual crate name. As someone who is not always focused on Iroha repository, I find it confusing time and time again to deal with.

Reply:
> And this will create so many conflicts for PRs

Well that's true.